src/Host.cpp

```diff
@@ -36,10 +36,11 @@
 #include "mudlet.h"
 
 #include "pre_guard.h"
-#include <QtUiTools>
 #include <QApplication>
 #include <QDir>
 #include <QMessageBox>
+#include <QStringBuilder>
+#include <QtUiTools>
```

Comment:
The current clang-format rules sort Qt* before Q*, so this will just get changed back.